docs/testnet_config.json.sample

```diff
@@ -1,6 +1,7 @@
 {
   "Configuration": {
     "ActiveNet": "testnet",
+    "Magic":2018111,
```

Comment:
Default config missing "magic" item here becase we have the default value already (default value is 2018101, please note this is the right magic value for testnet).
This sample config only list some essential items, those iterms with default values shall not be writen.

Reply:
Note that recently, magic number was changed for the testnet and that's the issue Coranos was having. And because no one was notified of this change, his height stopped growing suddently. After talking to Bocheng, he mentioned that the magic number had changed to 2018111 temporarily. This is what the PR is for to account for this change no matter how temporary it may be.

Reply:
Ok, I confirmed with Bocheng about that :)